File: resources/expected/ros2/building_control_gen_mixed_inverted_topics/strict/src/building_control_cpp_pkg/src/base_code/tcp_fan_runner.cpp
#include "building_control_cpp_pkg/user_headers/tcp_fan_src.hpp"

//========================================================
// Re-running Codegen will overwrite changes to this file
//========================================================

tcp_fan::tcp_fan() : tcp_fan_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("tcp_fan infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<tcp_fan>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-building_control_gen_mixed/strict/src/building_control_py_pkg/building_control_py_pkg/base_code/tcp_fan_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from building_control_py_pkg.user_code.tcp_fan_src import initialize
from building_control_py_pkg.base_code.tcp_fan_base_src import tcp_fan_base
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================

class tcp_fan(tcp_fan_base):
    def __init__(self):
        super().__init__()
        # invoke initialize entry point
        initialize(self)

        self.get_logger().info("tcp_fan infrastructure set up")

def main(args=None):
    rclpy.init(args=args)
    node = tcp_fan()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()
